src/renderer/app/index.css

```diff
@@ -69,3 +69,16 @@ ul:focus-visible {
 .scrollbar-stable {
   scrollbar-gutter: stable both-edges;
 }
+
+.visually-hidden {
```

Comment:
Well known styles that "hide" element, really useful with label + input (as instance: type="file")